lib/runAll.js

```diff
@@ -77,7 +77,7 @@ export const runAll = async (
     quiet = false,
     relative = false,
     shell = false,
-    stash = true,
+    stash = diff === undefined,
```

Comment:
To make this default more clear, let's add a small comment:

```suggestion
    // Stashing should be disabled by default when the `diff` option is used
    stash = diff === undefined,
```